apps/qubit/modules/informationobject/actions/editAction.class.php

```diff
@@ -816,4 +817,26 @@ private function incrementMaskCounter()
     $counter->value++;
     $counter->save();
   }
+
+  /**
+   * Trigger a job to update inherited fields in descendant records. Flash job started message to user.
+   */
+  private function updateDescendantInheritedFields()
+  {
+    if ($this->resource->rgt - $this->resource->lft <= 1)
+    {
+      return;
+    }
+
+    $job = QubitJob::runJob('arUpdateDescendantsJob', array('tldId' => $this->resource->id));
+
+    $params = array(
+      '%1%' => sprintf('<a href="%s">', $this->context->routing->generate(null, array('module' => 'jobs', 'action' => 'report', 'id' => $job->id))),
+      '%2%' => $job->id,
+      '%3%' => '</a>'
+    );
+
+    $message = $this->context->i18n->__('Updating inherited fields for descendants. Check %1%job %2%%3% to view the status of the job.', $params);
+    $this->getUser()->setFlash('notice', $message);
+  }
```

Comment:
I still think the job should be triggered from the IO save, as we now have an 'update on match' type of import and other places that may be making changes to the IOs.